Podfile.lock

```diff
@@ -2,13 +2,19 @@ PODS:
   - MockingbirdFramework (0.11.0)
```

Comment:
We need to commit podfile.lock because that's how we ensure everyone is using the same version of the libraries. So don't ignore this

Reply:
Yeah, lol I literally just changed this as you commented. I saw that on the cocoa-of documentation to always commit the pod file.lock